### PR TITLE
Buildscript & Unused handlers

### PR DESCRIPTION
**Pull request by mitchej123**
Upgrade buildscript
Nuke unused handlers
LGPL

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    compileOnly("curse.maven:cofh-core-69162:2388751") {
+        transitive = false
+    }
+    compileOnly("com.github.GTNewHorizons:Applied-Energistics-2-Unofficial:master-SNAPSHOT:dev") {
+        transitive = false
+    }
+    compileOnly("com.mod-buildcraft:buildcraft:7.1.23:dev") {
+        transitive = false
+    }
+
```
Comment (SinTh0r4s) [inline dependencies.gradle]: Empty line

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,60 @@
+modName = NEIIntegration
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = neiintegration
+customArchiveBaseName = NEIIntegration
```
Comment (SinTh0r4s) [inline gradle.properties:5]: Please keep the file structure and comments from ExampleMod. This way it is much easier to navigate the configuration files. Also, people don't read manuals or anything like that. The comment has the best chance of being read. It should be kept!

File: src/main/resources/mcmod.info [hunk 3]
```diff
@@ -3,8 +3,8 @@
   "modid": "neiintegration",
```
Comment (SinTh0r4s) [inline src/main/resources/mcmod.info:3]: "${modId}" and "${modName}" should be used here as well.

**Review by SinTh0r4s — DISMISSED** [review]
Is there any particular reason to not use the unified CI? I'd be nice to have the same structure and files everywhere.

Did we decide to kick out all integrations to mods not in the pack? Is there a list of "will never be added" mods? There are a lot more mods that would profit from this trimming!

**Comment by SinTh0r4s** [comment]
The `jitpack.yml` is missing

**Comment by bombcar** [comment]
> 

NEI is a special case in that @mitchej123 tries to keep it working for others. Not sure it applies to this. 

I believe this was literally thrown together last night in an attempt to memory hole the rock crusher. It can be “migrated” to the one true buildscript later.

**Comment by mitchej123** [comment]
> Is there any particular reason to not use the unified CI? I'd be nice to have the same structure and files everywhere.
> 
> Did we decide to kick out all integrations to mods not in the pack? Is there a list of "will never be added" mods? There are a lot more mods that would profit from this trimming!

This was done to remove handlers not used in the pack, such as the rock crusher.  Anyone else is free to use the upstream mod from curse.


As for the ci, I had nei open so I copied from there.  I wanted something buildable :)

**Comment by bombcar** [comment]
@SinTh0r4s we can merge this and then I can apply the rest of the build script after.

File: gradle.properties (hunk 2 above)
Comment (mitchej123) [reply gradle.properties:5]: It was copied from NEI and added this to keep the output file the same.

**Review by bombcar — APPROVED** [review]
will patch to match

**Pull request merged**